src/contracts/Staking.sol

```diff
@@ -30,28 +30,26 @@ contract Staking is Ownable {
     address public immutable COOL_DOWN_CONTRACT;
 
     // owner overrides
-    bool public pauseStaking = false;
-    bool public pauseUnstaking = false;
-    bool public overrideCanWithdraw = false;
+    bool public pauseStaking = false; // pauses staking
+    bool public pauseUnstaking = false; // pauses unstaking
 
-    // TODO: tightly pack for gas optimization
+    // tightly pack for gas optimization
```

Comment:
can probably remove this if you aren't going to try and pack them.

For reference - 

https://fravoll.github.io/solidity-patterns/tight_variable_packing.html